docs/workloads/azure/data/getting_started/core_data_platform_deployment_azure.md

```diff
@@ -7,12 +7,82 @@ hide_table_of_contents: false
 description: Infrastructure deployment
 keywords:
   - stacks cli
+  - data
+  - infrastructure
+  - azure
+  - template
 ---
 
-Azure DevOps pipelines for building and deploying the core infrastructure are configured in the
-[build](https://github.com/amido/stacks-azure-data/tree/main/build) folder.
+This section provides an overview of generating a new Data Platform project and deploying core infrastructure for Stacks Data Platform.
+This aligns to the workflow shown in the [deployment architecture](../architecture/architecture_data_azure.md#data-engineering-workloads) section.
+It assumes all prerequisites are in place, including:
 
-## TODO
+* [Azure](../requirements_data_azure.md#azure)
+* [Azure Pipelines variable groups](../requirements_data_azure.md#azure-pipelines-variable-groups)
 
-Documentation on using Stacks CLI to deploy core infrastructure components.
-Commands, configurations, requirements.
+## Step 1: Create/Generate Data Platform project using Stacks CLI
+
+The [Stacks CLI](../../../../stackscli/about) will help you get started with scaffolding your applications and workspaces using Stacks. Through a series of questions the CLI will determine how and what to build for your workspace, helping to accelerate your development process.
+
+Download and install the `stacks-cli` using [Stacks CLI](../../../../stackscli/about) page. Please refer to the **Stacks.CLI.Manual** in the latest `stacks-cli` release for detailed instruction.
+
+To construct a Data Platform project, two primary cli commands are required: `stacks-cli interactive` and `stacks-cli scaffold`.
+
+The interactive command is designed to ask questions on the command line about the configuration
+required for setting up Ensono Digital Stacks. It will then save this configuration out to a file that can be
+read in using the scaffold command.
+
+```cmd
+stacks-cli interactive
+```
+
+The majority of the questions are self-explanatory; please refer to the **Stacks.CLI.Manual** for further detail, however the following two will define the type of the target project. 
+
+| Question                                      | Required value for data project |
+|-----------------------------------------------|---------------------------------|
+| What framework should be used for the project?| infra                           |
+| Which type of infrastructure is required?     | data                            |
+
+The resulting configuration file named stacks.yml contains all of the configuration that was used to generate the projects,
+which means it can be used to produce the same project stack again.
+
+The CLI can be used with a configuration file to generate the Ensono Digital Stacks based projects using `stacks-cli scaffold`.
+
+```cmd
+stacks-cli scaffold -c ./stacks.yml
+```
+
+Open the project locally and push the generated project to target repository in `main/master` branch.
```

Comment:
Updated based on adding an assumption above that `main` is used in the new repo.
```suggestion
Open the project locally and push the generated project to the target remote repository's `main` branch.
```